src/index.d.ts

```diff
@@ -5226,6 +5226,7 @@ declare namespace admin.machineLearning {
     readonly sizeBytes: number;
 
     readonly gcsTfliteUri?: string;
+    readonly automlModelId?: string;
```

Comment:
Just noticed that the type name is `TFLiteModel` whereas the field is `gcsTfliteUri`. We probably should be consistent.